projects/WeTek_Play/options

```diff
@@ -111,13 +111,13 @@
     KODIPLAYER_DRIVER="libamcodec"
 
   # Modules to install in initramfs for early boot
-    INITRAMFS_MODULES="softcursor bitblit font fbcon"
+    INITRAMFS_MODULES=""
 
   # additional drivers to install:
   # for a list of additinoal drivers see packages/linux-drivers
   # Space separated list is supported,
   # e.g. ADDITIONAL_DRIVERS="DRIVER1 DRIVER2"
-    ADDITIONAL_DRIVERS=""
+    ADDITIONAL_DRIVERS="RTL8192CU RTL8192DU RTL8188EU"
```

Comment:
this i added because i removed the intree drivers (see your comment above). This in-tree drivers are basically the same drivers (+ some more). we have in this way a better control with patches which ads new product ids and patches etc...
All of the RTLxxxxXU drivers (with "U") are drivers for USB hardware.